filters/flowid/hashing.go

```diff
@@ -0,0 +1,37 @@
+package flowid
+
+import (
+	"crypto/rand"
+	"encoding/hex"
+	"errors"
+	"fmt"
+	"regexp"
+)
+
+const (
+	defaultLen = 16
+	maxLength  = 254
+	minLength  = 8
+)
+
+var (
+	ErrInvalidLen  = errors.New(fmt.Sprintf("Invalid length. len must be >= %d and < %d", minLength, maxLength))
+	flowIdRegex, _ = regexp.Compile(`^[\w+/=\-]+$`)
```

Comment:
in these cases, you can use MustCompile. it doesn't return an error, but panics immediately when running the tests. The idea is the same, the expression is not coming from outside, but you're one step safer by a panic during the tests.